internal/configuration/settings/wireguard.go

```diff
@@ -44,6 +44,9 @@ type Wireguard struct {
 	// It defaults to "auto" and cannot be the empty string
 	// in the internal state.
 	Implementation string `json:"implementation"`
+	// PersistenKeepAlive enable sending keep alive signal to server.
+	// So NAT view it as active connection
+	PersistenKeepAlive int `json:"persistent_keepalive"`
```

Comment:
let's make this a `*time.Duration` directly here and then you can use s.env.Duration to parse it for env variables.

Also add to the setDefaults method to set it to zero if it's nil, and add the field to the copy, mergeWith etc. methods below.